src/partition/ProvidedBoundingBox.cpp

```diff
@@ -208,8 +217,8 @@ void ProvidedBoundingBox::communicate()
   // each rank sends its mesh partition to connected remote ranks
   _m2ns[0]->broadcastSendLocalMesh(*_mesh);
   
-  createOwnerInformation();  
-  computeVertexOffsets();
+  createOwnerInformation();    
+  // computeVertexOffsetsBB(_mesh->vertices().size());    
```

Comment:
Why don't you use this function here?

Reply:
I have deactivated it to test! Forgot to re-activate! 
Thanks for reminding!